### ISSUE TITLE
[BUG]: Many powershell code items failed to run if privacy-script.bat is called with "setlocal EnableDelayedExpansion"

### ISSUE THREAD

**Issue by Mark-Joy** [undergroundwires/privacy.sexy#221]
### Description
If calling `privacy-script.bat` from another script which has `setlocal EnableDelayedExpansion`, many powershell code items failed to run.

### Reproduction steps
Create a script that call privact-script.bat as below:
```
@echo off & setlocal EnableDelayedExpansion
call  privact-script.bat
```

### Additional information
It can be fixed by adding "setlocal DisableDelayedExpansion" at the beginning of the privact-script.bat, (also possibly "EnableExtensions" for user convenience), so it would be:

```@echo off & setlocal EnableExtensions DisableDelayedExpansion```

**Comment by undergroundwires** [comment]
Thank you for bringing this issue, and providing detailed information on the bug and a possible solution. Good contribution and I will add this.

An alternative solution would be calling `DisableDelayedExpansion` on the caller side:

```batchfile
setlocal EnableDelayedExpansion 
   call  privact-script.bat
endlocal
```

However, I think  it is a good practice to manage the environment in the script privacy.sexy to make it as robust and independent as possible. The script itself should handle this without requiring users to know about the internal workings and requirements. So I will add this.

I believe we should also add `endlocal` at the end of the script to ensure this script does not interfere with callers environment.

Your future feedback is appreciated.

Start of the script:

```batchfile
@echo off
:: {{ $homepage }} — v{{ $version }} — {{ $date }}
:: Initialize script environment
setlocal EnableExtensions DisableDelayedExpansion
:: Ensure admin privileges
fltmc >nul 2>&1 || (
    echo Administrator privileges are required.
    PowerShell Start -Verb RunAs '%0' 2> nul || (
        echo Right-click on the script and select "Run as administrator".
        pause & exit 1
    )
    exit 0
)
```

End of the script:

```batchfile
endlocal
pause
exit /b 0
```

**Out-of-topic**: `pause` and `exit /b 0` would probably be intrusive as well. We could add ability to disable those lines with a flag, but this we should discuss in another github issue as I'd consider this a feature rather than a bug fix.

**Comment by undergroundwires** [comment]
Fixed and released in [0.12.6](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.12.6) 🚀
